dipy/segment/featurespeed.pxd

```diff
@@ -1,5 +1,5 @@
 from cythonutils cimport Data2D, Shape
-
+cimport numpy as np
```

Comment:
I personally prefer 'cimport numpy as cnp' so it is clearer which is Python numpy, which Cython.